katsdpcontroller/generator.py

```diff
@@ -409,7 +409,8 @@ def _make_fgpu(
         fgpu.cores = ['src0', 'src1', 'dst', 'python']
         if not configuration.options.develop:
             fgpu.numa_nodes = 1.0  # It's easily starved of bandwidth
-        fgpu.ports = ['port', 'prometheus']
+        fgpu.ports = ['port', 'prometheus', 'aiomonitor', 'aioconsole']
```

Comment:
Given all these new ports, it may be worth renaming `port` to something like `katcp`, unless there's already a convention on katsdpcontroller which should be followed.

Reply:
It's already a convention in katsdpcontroller. With SDP tasks the port mapping are written into the telstate config for the task and all the SDP containers look for it under that name, so changing it would be a pretty major exercise that would need lockstep changes across multiple repos.

It is a naming choice I regret though.